opentelemetry-sdk/src/opentelemetry/sdk/_logs/_internal/__init__.py

```diff
@@ -57,6 +58,13 @@
 _ENV_VALUE_UNSET = ""
 
 
+class LogDroppedAttributesWarning(UserWarning):
+    pass
+
+
+warnings.simplefilter("once", LogDroppedAttributesWarning)
```

Comment:
Isn't this the default?

Reply:
The `default` prints the first occurrence of matching warnings for each location (module + line number), while `once` prints only the first occurrence regardless of location. The problem with `once` is that we won't reissue the warning for the rest of the application's lifecycle. 

But I'm okay with the default behavior because one warning per LogRecord is also acceptable, although I think it's still excessive.